### DISCUSSION TITLE
100000x Scaleup

### DISCUSSION THREAD

**Discussion by divyang02** [divyang02/WolfLease#35] [Announcements]
Hi Team,

After implementing the frontend, integrating docker, and connecting the frontend and backend with a usable deployment I think we have achieved a great amount of scaleup. We were able to get a complete usable flow out of just barebones API which is great.

We are also running the frontend server on nginx which is really great for scalability. 

Awesome work folks

**Comment by bhansaliyash** [comment]
Got to learn a lot of new technologies.. Great work guys!!

**Comment by devmehta1999** [comment]
Great work guys. I would also like to mention the use of postman as well for testing the api's. Learning and growing together.!!

**Comment by gargpriyam21** [comment]
We were able to solve multiple issues in correlation to learning new things. Awesome work 😃.
